### ISSUE TITLE
[Bug]: Tooltip for login policy "Username and Password" is wrong - passwordless login only not possible

### ISSUE THREAD

**Issue by mffap** [zitadel/zitadel#8995]
### Preflight Checklist

- [x] I could not find a solution in the documentation, the existing issues or discussions
- [ ] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

ZITADEL Cloud

### Database

None

### Describe the problem caused by this bug

![Image](https://github.com/user-attachments/assets/a54887ea-452c-4e09-a2c7-9dc419782601)

The tooltip states that when disabling "Username and Password allowed" login with passwordless authentication is possible. In fact that's not the case at the moment, you can only login with an external IdP.

### To reproduce

1. Configure Passwordless
2. Disable Username and Password allowed

--> you're locked out of your account

### Expected behavior

The description should reflect the actual implementation.

**Comment by hifabienne** [comment]
@livio-a @muhlemmer @peintnermax from what i remember, the username password does only disable username password, and therefore allow external login, but i think passkey is not included here?
Please let me know if thats correct, in that case i can change the label so it is stated correct for the moment. 

Or is that something that also changes from the current to the new login?

**Comment by peintnermax** [comment]
Passwordless login is available under the condition that it is allowed in the loginsettings. Same for IDPs where we use the setting `External Login Allowed`. The tooltip should state: "Allows username and password authentication. Check your other settings to ensure users are not locked out" to avoid confusion.
The new login will adapt the actual state of the login settings.

**Comment by martinord** [comment]
Currently, if both these are disabled:
 
- `External Login allowed` (UNCHECKED)
- `Username and Password allowed` (UNCHECKED)

-> Then the passwordless login (via WebAuthn) cannot be used. There is no button, and I get the error:

`"Errors.Org.IdpNotExisting (LOGIN-5Hm8s)"`

Is this the expected behavior? Maybe this is related to #8996

**Comment by hifabienne** [comment]
> Currently, if both these are disabled:
> 
> * `External Login allowed` (UNCHECKED)
> * `Username and Password allowed` (UNCHECKED)
> 
> -> Then the passwordless login (via WebAuthn) cannot be used. There is no button, and I get the error in the login UI:
> 
> `"Errors.Org.IdpNotExisting (LOGIN-5Hm8s)"`
> 
> Is this the expected behavior? Maybe this is related to [#8996](https://github.com/zitadel/zitadel/issues/8996)

Yes in our currently login this is expected behaviour. We changed that for our newly implemented login v2, which is currently in Beta. The new login is in this repository: https://github.com/zitadel/typescript
And if you want to test it you can read everything on our docs: https://zitadel.com/docs/guides/integrate/login/hosted-login#hosted-login-version-2-beta

**Comment by vai** [comment]
Have also bumped into this with self-hosted. What's the fix given I'm locked out? I can access the DB, cli, etc. Not sure of the version I was running at the time, NixOS managed.

**Comment by rajatcing** [comment]
hey @vai https://zitadel.com/docs/apis/resources/mgmt/management-service-reset-login-policy-to-default you can reset the login settings and it should be fine again
